src/main/java/picard/arrays/illumina/Build37ExtendedIlluminaManifestRecord.java

```diff
@@ -40,9 +40,16 @@
         ILLUMINA_FLAGGED,   // Illumina flagged
         LIFTOVER_FAILED,
         UNSUPPORTED_GENOME_BUILD,
-        SEQUENCE_MISMATCH,      // mismatch between what the manifest claims is the reference vs. the actual reference.
-        INDEL_SEQ_MISMATCH,     // Unable to reconcile indel situation
-        INDEL_EXTENSION_ERROR,  // extension base conflict.
+        PROBE_SEQUENCE_MISMATCH,        // Probe sequence not found in reference.
```

Comment:
It would be better to use javadoc here instead of `//` comments, e.g.
```
/** Probe sequence not found in reference. */
PROBE_SEQUENCE_MISMATCH,
```

Reply:
Yup